cdap-ui/app/features/flows/controllers/tabs/runs/tabs/status-ctrl.js

```diff
@@ -27,12 +27,12 @@ angular.module(PKG.name + '.feature.flows')
     }
 
     function pollMetrics() {
-      if (!$scope.runs.length) {
-        return;
-      }
       var nodes = $scope.data.nodes;
       // Requesting Metrics data
       angular.forEach(nodes, function (node) {
```

Comment:
Previously, we were not hitting metrics for streams, if the flow had no runs. That was improper.